code/modules/food_and_drinks/pizzabox.dm

```diff
@@ -44,8 +44,21 @@
 	update_appearance()
 	register_context()
 
+/obj/item/pizzabox/proc/register_bomb(new_bomb)
+	bomb = new_bomb
+	if(istype(bomb))
+		RegisterSignal(bomb, COMSIG_PARENT_QDELETING, PROC_REF(clear_bomb))
+
+/obj/item/pizzabox/proc/clear_bomb(datum/source)
+	SIGNAL_HANDLER
+	if(!bomb)
```

Comment:
```suggestion
	if(isnull(bomb))
```

or even
```suggestion
	if(QDELETED(bomb))
```

just because it's a tad bit more explicit in this scenario (and `isnull()` is faster than `!`)

Reply:
If it's `QDELING` (which `QDELETED` includes) we should absolutely clear the reference, though. I'll go with `isnull`.